### PR TITLE
Clueboard Rev4 Updates aka Volcano 660

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Went to go play around in the code and add some of my own things, realized things were a little different here. 

Things I changed:

~~1. pragma once the files that are missing it~~ EDIT: zvecr says NO
2. Turn on bootmagic lite. The Volcano 660 has a solid bottom so if a user forgets to put on a RESET key in their keymap, it's back to dismantling
3. RGBLIGHT is enabled.....even though there is no RGB light on this board
4. BACKLIGHT is enabled.....even though there is no backlight on this board
5. Enabled COMMAND_ENABLE for extra debugging prowess
6. AUDIO_ENABLE was included twice, once disabling and once enabling. Assuming its enabling as why else would you have a speaker? 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clueboard/66/rev4/rev4.c [hunk 1]
```diff
@@ -12,35 +11,6 @@ void matrix_scan_kb(void) {
...
-        PORTD &= ~(1<<6); // Esc
-        PORTB &= ~(1<<7); // Page Up
-        PORTD &= ~(1<<4); // Arrows
-    }
-*/
-}
-
 void led_init_ports() {
     // Set our LED pins as output
     palSetPadMode(GPIOB, 13, PAL_MODE_OUTPUT_PUSHPULL); // LED1
```
Comment (fauxpark) [inline keyboards/clueboard/66/rev4/rev4.c]: These `PUSHPULL` lines can be changed to eg.
```suggestion
    setPinOutput(B13); // LED1
```
And the `palClearPad()`s to `writePinLow()` etc.

File: keyboards/clueboard/66/rev4/chconf.h [hunk 2]
```diff
@@ -25,9 +25,6 @@
  * @{
  */
 
-#ifndef CHCONF_H
```
Comment (zvecr) [inline keyboards/clueboard/66/rev4/chconf.h]: These should be put back, as its now running with no guards whatsoever. (Im hesitant to suggest pragma once as its interacting with ChibiOS and thats kinda a black box).

File: keyboards/clueboard/66/rev4/mcuconf.h [hunk 3]
```diff
@@ -14,9 +14,6 @@
     limitations under the License.
 */
 
-#ifndef MCUCONF_H
```
Comment (zvecr) [inline keyboards/clueboard/66/rev4/mcuconf.h]: same as chconf

**Comment by mechmerlin** [comment]
All comments resolved!

**Pull request merged**